index.js

```diff
@@ -12,3 +12,177 @@ document.querySelector('.popup-menu .menu-item').addEventListener('click', close
 document.querySelector('.popup-menu .menu-item2').addEventListener('click', closePopUp);
 document.querySelector('.popup-menu .menu-item3').addEventListener('click', closePopUp);
 document.querySelector('.popup-nav').addEventListener('click', closePopUp);
+document.querySelector('.popup-nav');
+
+// Project Cards
+
+const ProjecData = [
+  {
+    id: 1,
+    name: 'Tonic',
+    company: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    techStacks: ['html', 'css', 'javaScript'],
+    FeaturedDesktopImage: 'images/Portoflio Card/Snapshoot Portfolio.png',
+    FeaturedMobileImage: 'images/Snapshoot Portfolio.png',
+    liveVersionLink: '#',
+    sourceLink: '#',
```

Comment:
I'd suggest updating the `sourceLink` and `liveVersionLink`  in your objects so that when the button is clicked it redirects to a real link 🔗 it can be anything, just for the purpose of making it work.  😉